### PR TITLE
Introduce iExec Sidechain

### PR DESCRIPTION
**Pull request by zguesmi**
* Add an Overview page.
* Add instructions to connect to the Sidechain.

### CODE REVIEW

File: for-developers/sidechain/overview.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# iExec sidechain
+iExec sidechain, aka **Bellecour**, is a Proof of Authority Blockchain that aims at scaling the underling infrastructure of iExec's off-chain/confidential computing platform. It is linked to Ethereum Mainnet with a bridge that enables sending [RLC](https://iex.ec/rlc-token/) tokens in both directions.
+
+* **Explorer**: <https://blockscout.bellecour.iex.ec>
+* **Netstats**: <https://netstats.bellecour.iex.ec>
+* **Governance Dapps**:
```
Comment (ericr6) [inline for-developers/sidechain/overview.md]: should we move those links to Validators section?

File: for-developers/sidechain/overview.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Validators
+Currently, the Sidechain is running with 8 validators, some of them are not controlled by iExec which makes the chain more decentralized and resilient.
+
+## Whitelisting
+By default, all operations related to iExec's Marketplace such as deploying Dapps/Datasets/Workerpools and running tasks are permissable without any authorization. However, developers who want to deploy custom Smart Contracts need to send a whitelisting request using the following [link](https://iexecproject.atlassian.net/servicedesk/customer/portal/4/group/9/create/66).
+
+## Bridge
+Tasks on iExec's off-chain/confidential computing marketplace are paid with ERC20 RLC tokens. Users wanting to interact with the platform need to bridge RLC tokens from Ethereum Mainnet to the Sidechain. The bridged asset is called xRLC on Bellecour.
+```
+1 RLC = 1 xRLC
```
Comment (ericr6) [inline for-developers/sidechain/overview.md:31]: Should we provide more information about what XRLC is? XRLC represents 1 RLC locked on the mainnet, and there is no magical issuance involved.

File: for-developers/sidechain/overview.md (hunk 1 above)
Comment (zguesmi) [reply for-developers/sidechain/overview.md]: Good idea yes :+1:

File: for-developers/sidechain/overview.md (hunk 2 above)
Comment (zguesmi) [reply for-developers/sidechain/overview.md:31]: Done.

File: for-developers/sidechain/overview.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# iExec sidechain
+iExec sidechain, aka **Bellecour**, is a Proof of Authority Blockchain that aims at scaling the underling infrastructure of iExec's off-chain/confidential computing platform. It is linked to Ethereum Mainnet with a bridge that enables sending [RLC](https://iex.ec/rlc-token/) tokens in both directions.
+
+* **Explorer**: <https://blockscout.bellecour.iex.ec>
+* **Netstats**: <https://netstats.bellecour.iex.ec>
+
+## Key features & highlights
+* **EVM-compatibility**: Bellecour is an EVM-compatible chain so there is no need to modify contracts in order to deploy them the sidechain.
```
Comment (ericr6) [inline for-developers/sidechain/overview.md]: deploy them on the sidechain

File: for-developers/sidechain/overview.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## Validators
+Currently, the Sidechain is running with 8 validators, some of them are not controlled by iExec which makes the chain more decentralized and resilient.
+The governance of the chain is managed through ballots and votes.
+#### Governance Dapps:
+* Voting: <https://voting.bellecour.iex.ec>
+* Validators: <https://validator.bellecour.iex.ec>
+* Ceremony: <https://ceremony.bellecour.iex.ec>
+
+## Whitelisting
+By default, all operations related to iExec's Marketplace such as deploying Dapps/Datasets/Workerpools and running tasks are permissable without any authorization. However, developers who want to deploy custom Smart Contracts need to send a whitelisting request using the following [link](https://iexecproject.atlassian.net/servicedesk/customer/portal/4/group/9/create/66).
```
Comment (ericr6) [inline for-developers/sidechain/overview.md]: permissable => allowed

File: for-developers/sidechain/connect.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+![Add iExec Sidechain using Chainlist](../../.gitbook/assets/chainlist.png)
+
+## Option 2 - Blockscout
+iExec Sidechain could also be added to wallets directly from its explorer <https://blockscout.bellecour.iex.ec>.
+
+![Add iExec Sidechain using Blockscout](../../.gitbook/assets/blockscout.png)
+
+## Option 3 - iExec account manager
+iExec account manager present across iExec products, when used for the first time, allows users to add iExec Sidechain to the network list of their wallets.
```
Comment (ericr6) [inline for-developers/sidechain/connect.md]: iexec account manager is available across multiple iExec Products

File: for-developers/sidechain/overview.md (hunk 3 above)
Comment (zguesmi) [reply for-developers/sidechain/overview.md]: Done.

File: for-developers/sidechain/overview.md (hunk 4 above)
Comment (zguesmi) [reply for-developers/sidechain/overview.md]: Done.

File: for-developers/sidechain/connect.md (hunk 5 above)
Comment (zguesmi) [reply for-developers/sidechain/connect.md]: Done.

**Review by ericr6 — APPROVED** [review]

**Pull request merged**